src/core/transport/transport.h

```diff
@@ -139,8 +139,10 @@ typedef struct grpc_transport_op {
   gpr_slice *goaway_message;
   /** set the callback for accepting new streams;
       this is a permanent callback, unlike the other one-shot closures */
```

Comment:
the docstring seems misplaced with the introduction of the boolean guard (which isn't mentioned).